cmd/lukso/dependencies.go

```diff
@@ -41,111 +43,114 @@ var (
 			baseUrl:  "https://gethstore.blob.core.windows.net/builds/geth-%s-amd64-%s-%s.tar.gz",
 			name:     gethDependencyName,
 			filePath: "", // binary dir selected during runtime
+			isBinary: true,
 		},
 		prysmDependencyName: {
 			baseUrl:  "https://github.com/prysmaticlabs/prysm/releases/download/%s/beacon-chain-%s-%s-amd64",
 			name:     prysmDependencyName,
 			filePath: "", // binary dir selected during runtime
+			isBinary: true,
 		},
 		validatorDependencyName: {
 			baseUrl:  "https://github.com/prysmaticlabs/prysm/releases/download/%s/validator-%s-%s-amd64",
 			name:     validatorDependencyName,
 			filePath: "", // binary dir selected during runtime
+			isBinary: true,
 		},
 
 		// ----- CONFIGS -----
 		// ----- MAINNET -----
 		gethMainnetGenesisDependencyName: {
 			baseUrl:  "https://raw.githubusercontent.com/lukso-network/network-configs/main/mainnet/shared/genesis.json",
 			name:     gethMainnetGenesisDependencyName,
-			filePath: "./config/mainnet/shared/genesis.json",
+			filePath: "./configs/mainnet/shared/genesis.json",
 		},
 		gethMainnetConfigName: {
 			baseUrl:  "https://raw.githubusercontent.com/lukso-network/network-configs/main/mainnet/geth/geth.toml",
 			name:     gethMainnetConfigName,
-			filePath: "./config/mainnet/geth/geth.toml",
+			filePath: "./configs/mainnet/geth/geth.toml",
```

Comment:
might be nice maybe to have a prefix at the top: `configs/` and then reuse it everywhere